src/shared/sources/us/wa/index.js

```diff
@@ -0,0 +1,213 @@
+    },
+    {
+      startDate: '2020-03-19',
+      crawl: [
+        {
+          type: 'headless',
+          data: 'table',
+          url: 'https://www.doh.wa.gov/Emergencies/Coronavirus'
+        }
+      ],
+      scrape ($) {
+        let counties = []
+        const $table = $('caption:contains("Number of Individuals Tested")')
+          .first()
+          .closest('table')
+        const $trs = $table.find('tbody > tr')
+        $trs.each((index, tr) => {
+          const $tr = $(tr)
+          const cases = parse.number(parse.string($tr.find('td:nth-child(2)').text()) || 0)
+          const deaths = parse.number(parse.string($tr.find('td:last-child').text()) || 0)
+
+          let county = geography.addCounty(parse.string($tr.find('> *:first-child').text()))
+          if (county === 'Unassigned County') {
+            county = constants.UNASSIGNED
+          }
+          if (index < 1 || index > $trs.get().length - 2) {
+            return
+          }
+          counties.push({
+            county,
+            cases,
+            deaths
+          })
+        })
+        counties = geography.addEmptyRegions(counties, allCounties, 'county')
+        counties.push(transform.sumData(counties))
+        return counties
+      }
+    },
+    {
+      startDate: '2020-03-23',
+      crawl: [
+        {
+          type: 'headless',
+          data: 'table',
+          url: 'https://www.doh.wa.gov/Emergencies/Coronavirus'
+        }
+      ],
+      scrape ($) {
+        let counties = []
+        const $table = $('caption:contains("Confirmed Cases")')
+          .first()
+          .closest('table')
+
+        const $trs = $table.find('tbody > tr')
+        $trs.each((index, tr) => {
+          const $tr = $(tr)
+          const cases = parse.number(parse.string($tr.find('td:nth-child(2)').text()) || 0)
+          const deaths = parse.number(parse.string($tr.find('td:last-child').text()) || 0)
+
+          let county = geography.addCounty(parse.string($tr.find('> *:first-child').text()))
+          if (county === 'Unassigned County') {
+            county = constants.UNASSIGNED
+          }
+          if (county === 'Total County') {
+            return
+          }
+          counties.push({
+            county,
+            cases,
+            deaths
+          })
+        })
+        counties = geography.addEmptyRegions(counties, allCounties, 'county')
+        counties.push(transform.sumData(counties))
+        return counties
+      }
+    },
+    {
+      startDate: '2020-03-30',
+      crawl: [
+        {
+          type: 'json',
+          url: 'https://services8.arcgis.com/rGGrs6HCnw87OFOT/arcgis/rest/services/CountyCases/FeatureServer/0/query?f=json&where=(CV_State_Cases%3E0)&returnGeometry=false&outFields=*&orderByFields=CNTY_NAME%20asc'
+        }
+      ],
+      scrape (data, scrapeDate) {
+        const counties = []
+        data.features.forEach(item => {
+          const cases = item.attributes.CV_PositiveCases
```

Comment:
We should have a library for json schema validation.  That's not a blocker for this PR, though.